### ISSUE TITLE
Phantosmia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6776]
**Preferred term label:**
Phantosmia

**Synonyms**
Olfactory hallucination
Phantom odor

**Definition (free text, please give PubMed ID)**
Perception of an odor in the absence of any stimuli in the surrounding environment that could emit the odor. 
PMID:28334095

Phantosmia has been documented in individuals with epileptic seizures, schizophrenia, depression, migraine, and otorhinolaryngology problems. Phantosmia occurs in a variety of clinical conditions, and its causes are yet unknown. Prior accounts have suggested that the phantosmic sensations originate either in the peripheral olfactory nervous system or in central brain regions, such as the amygdala and the orbital frontal cortex. Patients who experienced phantosmia following head trauma were characterized by left frontal atrophy, suggestive of a cortical origin. However, a patient study reported successfully resolving phantosmia in 7 out of 8 patients through excision of the olfactory epithelium, suggesting also peripheral olfactory system involvement

**Parent term (use hpo.jax.org/app)**
Abnormality of the sense of smell HP:0004408

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@DLewisSmith this is a finding in persons with long COVID (https://www.medrxiv.org/content/10.1101/2020.12.24.20248802v2). We have previously characterized the "hallucination" terms as kinds of focal seizure, but I do not think that is generally the case here. I am cc'ing you in case there is anything that we should add from a neurological perspective, or if there is anything we can do to differentiate peripheral and central causes of phantosmia.

**Comment by DLewisSmith** [comment]
I think it could be reasonable to make ‘ Focal sensory seizure with olfactory features HP:0011161’ a child of this. I like your explicit definition that describes, but avoids the word, ‘hallucination’ as this is often misinterpreted.

**Comment by pnrobinson** [comment]
added, thanks!
